openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -226,16 +226,12 @@ def load_from_module_store(cls, course_id):
                         ])
                         CourseOverviewImageSet.create_or_update(course_overview, course)
 
-                except IntegrityError:
-                    # There is a rare race condition that will occur if
-                    # CourseOverview.get_from_id is called while a
-                    # another identical overview is already in the process
-                    # of being created.
-                    # One of the overviews will be saved normally, while the
-                    # other one will cause an IntegrityError because it tries
-                    # to save a duplicate.
-                    # (see: https://openedx.atlassian.net/browse/TNL-2854).
-                    pass
+                except Exception:  # pylint: disable=broad-except
```

Comment:
A broad except like this is not advisable since it is swallowing all sorts of exceptions - including those that should not allow continuing.  Instead, may I suggest changing the logic to:

```
try:
    # as today
except IntegrityError:
    # as today
    pass
except Exception:
    # log so we know what it is, but raise so flow is discontinued
    log.exception
    raise
```